UPDATING.md

```diff
@@ -421,7 +421,7 @@ indefinitely. This is only available on the command line.
 After how much time should an updated DAG be picked up from the filesystem.
 
 #### min_file_parsing_loop_time
-
+CURRENTLY DISABLED DUE TO A BUG
```

Comment:
Please add a note to the master section, we are not changing the past. People will read it as “I am at this version, what do I need to do to get to the next”

Reply:
Done

Reply:
Done, PTAL